libexec/alluxio-config.sh

```diff
@@ -46,15 +46,20 @@ fi
 JAVA_HOME=${JAVA_HOME:-"$(dirname $(which java))/.."}
 JAVA=${JAVA:-"${JAVA_HOME}/bin/java"}
 
-# Make sure alluxio-env.sh exists
-if [[ ! -e ${ALLUXIO_CONF_DIR}/alluxio-env.sh ]]; then
-  echo "Cannot find ${ALLUXIO_CONF_DIR}/alluxio-env.sh. To proceed, you can" >&2
-  echo "(1) create one based on the provided template file ${ALLUXIO_CONF_DIR}/alluxio-env.sh.template, or" >&2
-  echo "(2) use a bootstraping tool by running: ${ALLUXIO_HOME}/bin/alluxio bootstrapConf" >&2
-  exit 1
+# Determine reasonable defaults for worker memory and ramdisk folder
+if [[ $(uname -s) == Darwin ]]; then
+  # Assuming Mac OS X
+  DEFAULT_RAM_FOLDER="/Volumes/ramdisk"
+else
+  # Assuming Linux
+  DEFAULT_RAM_FOLDER="/mnt/ramdisk"
 fi
+# If ALLUXIO_RAM_FOLDER is explicitly set to the empty string, do not overwrite.
+ALLUXIO_RAM_FOLDER=${ALLUXIO_RAM_FOLDER-${DEFAULT_RAM_FOLDER}}
```

Comment:
actually, shall we follow the same procedure here for `ALLUXIO_RAM_FOLDER`, and `ALLUXIO_WORKER_MEMORY_SIZE`?

- Check if the env var has been explicitly set, if yes, we are good
- Otherwise, check if the corresponding site-properties set by `getConf`. if yes, we are good
- Otherwise, check the OS and choose the default value.